packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -25,27 +25,30 @@ const trace = makeTracer('IV');
 /**
  * @file This has most of the logic for a Vault, to borrow RUN against collateral.
  *
- * The logic here is for InnerVault which is the majority of logic of vaults but
- * the user view is the `vault` value contained in VaultKit.
+ *   The logic here is for InnerVault which is the majority of logic of vaults but
+ *   the user view is the `vault` value contained in VaultKit.
  *
- * A note on naming convention:
- * - `Pre` is used as a postfix for any mutable value retrieved *before* an
- *    `await`, to flag values that must used very carefully after the `await`
- * - `new` is a prefix for values that describe the result of executing a
- *   transaction; e.g., `debt` is the value before the txn, and `newDebt`
- *   will be value if the txn completes.
- * - the absence of one of these implies the opposite, so `newDebt` is the
- *   future value fo `debt`, as computed based on values after any `await`
+ *   A note on naming convention:
+ *
+ *   - `Pre` is used as a postfix for any mutable value retrieved _before_ an
+ *       `await`, to flag values that must used very carefully after the `await`
+ *   - `new` is a prefix for values that describe the result of executing a
+ *       transaction; e.g., `debt` is the value before the txn, and `newDebt`
+ *       will be value if the txn completes.
+ *   - the absence of one of these implies the opposite, so `newDebt` is the future
+ *       value fo `debt`, as computed based on values after any `await`
  */
 
 /**
  * Constants for vault phase.
  *
- * ACTIVE       - vault is in use and can be changed
- * LIQUIDATING  - vault is being liquidated by the vault manager, and cannot be changed by the user
- * TRANSFER     - vault is able to be transferred (payments and debits frozen until it has a new owner)
- * CLOSED       - vault was closed by the user and all assets have been paid out
- * LIQUIDATED   - vault was closed by the manager, with remaining assets paid to owner
+ * - ACTIVE - vault is in use and can be changed
+ * - LIQUIDATING - vault is being liquidated by the vault manager, and cannot be
+ *   changed by the user
+ * - TRANSFER - vault is able to be transferred (payments and debits frozen until
+ *   it has a new owner)
+ * - CLOSED - vault was closed by the user and all assets have been paid out
+ * - LIQUIDATED - vault was closed by the manager, with remaining assets paid to owner
```

Comment:
This is harder to read; is there a convention for vertical alignment in JSDoc?

Reply:
Good call. Yeah, instead of list items I should have just indented:
<img width="724" alt="Screen Shot 2022-03-24 at 7 05 08 AM" src="https://user-images.githubusercontent.com/21505/159933848-2e428209-6dae-4d58-b05f-2fe623209ac4.png">

c43b982cba82928eadfa5d18219d2284f29382cd

Reply:
I lost this change. I could restore it if necessary

Reply:
Looks fine now.